README.md

```diff
@@ -1,70 +1,143 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
+  <img src="./src/assets/logo.png" alt="logo" width="200"  height="auto" />
+  <br/>
 
-## Available Scripts
+  <h3><b>Space Travelers' Hub</b></h3>
 
-In the project directory, you can run:
+</div>
 
-### `npm start`
+# 📗 Table of Contents
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
```

Comment:
- Hey, since you've already added the tech stack used in building the project in the `Built with` section, please, kindly remove the `Tech Stack` to make the readme more professional.

Reply:
Hi @zieeco thanks for the comment. The Tech stack has been removed.